.env.development

```diff
@@ -14,7 +14,7 @@ DUTY_CALCULATOR=true
 DUTY_CALCULATOR_BASE_URL=http://localhost:3002/duty-calculator
 FRONTEND_HOST=http://localhost
 HTTPARTY_READ_TIMEOUT=661
-PERMUTATIONS=true
+PERMUTATIONS=false
 PORT=3001
 ROO_WIZARD=true
 SEARCH_BANNER=true
```

Comment:
I believe we'd discussed disabling the search banner in local development which is maybe what you'd intended to change

Reply:
I think this is a merging error

Reply:
I fixed it now